### PR TITLE
Improve client interop compatibility

### PR DESCRIPTION
**Pull request by Inphi**
This replaces prysm's interop mocks to use generated (pre-merge) genesis files. Doing this makes it easy to add new clients by relying only on the chain config and the genesis beacon state.

### CODE REVIEW

File: geth/geth-genesis.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "londonBlock": 0,
     "shardingForkBlock": 66,
     "clique": {
-      "period": 5,
+      "period": 14,
```
Comment (jimmygchen) [inline geth/geth-genesis.json:18]: probably need to update the deposit contract address to `0x4242424242424242424242424242424242424242` under `alloc` as well

Comment (Inphi) [reply geth/geth-genesis.json:18]: The geth genesis will be generated by `testnetgen` and we'll no longer need this file.

**Comment by Inphi** [comment]
Since lighthouse and a couple other clients now support starting from a bellatrix genesis beacon state, we can simplify this setup to avoid the merge.

**Comment by Inphi** [comment]
This idea has been adopted via #88

**Pull request closed without merging**